src/server/index.js

```diff
@@ -43,10 +43,11 @@ app.ws("/dominion", function(ws, req) {
   store.dispatch(asyncAddConnection({ ws, id: connectionId, username }));
 
   ws.on("message", function(msg) {
-    if (isEmpty(msg)) {
+    const msgObj = JSON.parse(msg);
+    if (isEmptyObject(msgObj)) {
```

Comment:
should be `isEmpty(msgObj)`, right? you can also run `npm run lint` to catch spelling mistakes

Reply:
Fixed